types/matter-js/index.d.ts

```diff
@@ -2039,6 +2040,7 @@ declare namespace Matter {
         * @property broadphase
         * @type grid
         * @default a Matter.Grid instance
+        * @deprecated use `engine.grid`
```

Comment:
This looks like it might have been deprecated in 0.17.  Does the version in the header need to be updated?

Reply:
I think so too, didn't see the version header. I'll go through the matter-js changelog and add the new Types since 14.2